app.js

```diff
@@ -1,128 +1,55 @@
-  },
-
-  // simulates the number of cookies bought in each hour of a work day
-  simCookiesHrs: function() {
-    this.cookiesHrs = [];
-    var cookiesBought;
-    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
-      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
-      this.cookiesHrs.push(cookiesBought);
-    }
-  },
+/** Returns a random number of customers for one hour at the store **/
+Store.prototype.randomCustHr = function() {
+  return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
 };
 
-var centerStore = {
-  location: 'Seattle Center',
-  openAt: 6, // 6 am
-  closeAt: 20, // 8 pm
-  minCustHr: 11,
-  maxCustHr: 38,
-  avgCookieCust: 3.7,
-  cookiesHrs: [],
-
-  // calculates random number of customers for one hour
-  randomCustHr: function() {
-    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
-  },
-
-  // simulates the number of cookies bought in each hour of a work day
-  simCookiesHrs: function() {
-    this.cookiesHrs = [];
-    var cookiesBought;
-    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
-      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
-      this.cookiesHrs.push(cookiesBought);
-    }
-  },
+/** Simulates the number of cookies bought in each hour of a work day **/
+Store.prototype.simCookiesHrs = function() {
+  this.cookiesHrs = [];
+  var cookiesBought;
+  for (var i = 0; i < (this.closeAt - this.openAt); i++) {
+    cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
+    this.cookiesHrs.push(cookiesBought);
+  }
 };
 
-var capitolStore = {
-  location: 'Capitol Hill',
-  openAt: 6, // 6 am
-  closeAt: 20, // 8 pm
-  minCustHr: 20,
-  maxCustHr: 38,
-  avgCookieCust: 2.3,
-  cookiesHrs: [],
-
-  // calculates random number of customers for one hour
-  randomCustHr: function() {
-    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
-  },
-
-  // simulates the number of cookies bought in each hour of a work day
-  simCookiesHrs: function() {
-    this.cookiesHrs = [];
-    var cookiesBought;
-    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
-      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
-      this.cookiesHrs.push(cookiesBought);
-    }
-  },
-};
+/** Given a table element reference, adds a tr element with the location name, all the simulated cookie data and the total for the day to the table **/
+Store.prototype.render = function(table) {
+  var cookiesDayRow = document.createElement('tr');
+  cookiesDayRow.setAttribute('class', 'sales-data');
+
+  var locationCell = document.createElement('td');
+  locationCell.setAttribute('class', 'loc-cell');
+  locationCell.textContent = this.location;
+  cookiesDayRow.appendChild(locationCell);
+
+  this.simCookiesHrs(); // activate simulation of cookies
```

Comment:
consider adding this to the constructor function.  Since this function generates hourly sales and render function generates tables, these two things should be decoupled.